### PR TITLE
Bump 6.0.0

### PR DESCRIPTION
**Pull request by sdvillal**
Also use new build/host/run and new compilers

### CODE REVIEW

**Comment by sdvillal** [comment]
@conda-forge/av This should be ready to merge now. Can you please review?

 I have disabled python 2 windows builds, as they were problematic. If that is not OK, I still suggest merging this and opening a new PR to work on it - especially given how long it takes these days for a windows job to run.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ build:
   number: {{ build_number }}
   skip: True  # [win and py27]
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  run_exports:
+    # PyAV now claims to follow semantic versioning
```
Comment (mikeboers) [inline recipe/meta.yaml]: Haha. We sure do claim that.

**Comment by mikeboers** [comment]
I've been asked to review, but to be honest I have no idea what I'm doing here.

From the context of someone who has never interacted with Conda at this level (or honestly for more than 20 minutes total), the changes that I see that connect with the context of PyAV development make sense to me.

I'm all for the idea of merging and then moving on. I'd personally rather have a slightly broken up-to-date one that we then get feedback on, then one that we are holding back out of fear.

(So, uh, who has that power? It looks like I do, but... yeah.)

**Comment by sdvillal** [comment]
Yeah, you have the power. Once this gets merged, I will have it too. I also would suggest to merge sooner rather than later

**Comment by mikeboers** [comment]
Great. Will do.

Bug me again tomorrow if I forget in the chaos of going to the office today.

**Comment by mikeboers** [comment]
Done.

Thank you for your work, and offering to take care of this package in the future.

**Comment by sdvillal** [comment]
Thanks Mike.

**Pull request merged**